waiter/integration/waiter/cookie_support_integration_test.clj

```diff
@@ -0,0 +1,42 @@
+;;
+;;       Copyright (c) 2017 Two Sigma Investments, LP.
+;;       All Rights Reserved
+;;
+;;       THIS IS UNPUBLISHED PROPRIETARY SOURCE CODE OF
+;;       Two Sigma Investments, LP.
+;;
+;;       The copyright notice above does not evidence any
+;;       actual or intended publication of such source code.
+;;
+(ns waiter.cookie-support-integration-test
+  (:require [clojure.data.json :as json]
+            [clojure.test :refer :all]
+            [waiter.client-tools :refer :all]))
+
+(deftest ^:parallel ^:integration-fast test-cookie-support
+  (testing-using-waiter-url
+    (let [extra-headers {:x-waiter-name (rand-name)
```

Comment:
Nitpick: can we make two separate `testing` fragments each with their own `x-kitchen-cookies` header.